src/fluent/watch.ts

```diff
@@ -97,6 +98,8 @@ export class Watcher<T extends GenericClass> {
   // Track the list of items in the cache
   #cache = new Map<string, InstanceType<T>>();
 
+  // Token Path
+  #TOKEN_PATH = "/var/run/secrets/kubernetes.io/serviceaccount/token";
```

Comment:
how does this work when running outside the cluster, ie `npx pepr dev`? dont love the hard-coding here

Reply:
if the token is null it won't be added to the [header](https://github.com/defenseunicorns/kubernetes-fluent-client/blob/88fb8fb57c6bbb676926f7661c15da511ceb9a2c/src/fluent/watch.ts#L444) but the client will have a [cert and key](https://github.com/defenseunicorns/kubernetes-fluent-client/blob/88fb8fb57c6bbb676926f7661c15da511ceb9a2c/src/fluent/watch.ts#L433) from the kube-config.